scripts/run_backend_tests.py

```diff
@@ -263,6 +263,28 @@ def _get_all_test_targets_from_shard(shard_name):
     return shards_spec[shard_name]
 
 
+def _check_shards_cover_all_tests(include_load_tests=True):
+    """Check whether the test shards cover all test classes.
+
+    Args:
+        include_load_tests: bool. Whether to include load tests.
+
+    Returns:
+        str. The dotted name of a test class not included in the shards,
+        or an empty string if all tests are included.
+    """
+    with python_utils.open_file(SHARDS_SPEC_PATH, 'r') as shards_file:
+        shards_spec = json.load(shards_file)
+    test_modules = tuple(
+        module for shard in shards_spec.values() for module in shard)
+    test_classes = _get_all_test_targets_from_path(
+        include_load_tests=include_load_tests)
+    for test_class in test_classes:
+        if not test_class.startswith(test_modules):
+            return test_class
```

Comment:
Also check:
1. Reverse of this: That all modules in the shards actually exist
2. For modules duplicated across shards

Reply:
Done in a76b8e2c1